svc-bie-kafka/src/main/java/gov/va/vro/services/bie/config/MessageExchangeConfig.java

```diff
@@ -28,11 +30,14 @@ public MessageConverter messageConverter() {
   @Bean
   Declarables topicBindings(final BieProperties bieProperties) {
     final List<AbstractDeclarable> list =
-        bieProperties.getTopicToExchangeMap().values().stream()
+        Arrays.stream(bieProperties.topicNames())
```

Comment:
Since Kafka topics are listed in ContentionEvent only (and not in application.yaml), this should iterate through the `ContentionEvent` enum values. Then, line 38 can be `contentionEvent.rabbitMqExchangeName()`.

Reply:
`topicNames` already gets values from `ContentionEvent`

Reply:
But it's using `bieProperties` which is not where the topics are.

Reply:
This code should be iterating over the Kafka topics and MQ exchange names provided by `ContentionEvent`. `bieProperties` shouldn't be used anywhere except in `ContentionEvent`.